code/__HELPERS/path.dm

```diff
@@ -143,8 +143,8 @@
 	if(!start || !end)
 		stack_trace("Invalid A* start or destination")
 		return
-	if(start.z != end.z || start == end ) //no pathfinding between z levels
-		return
+	// if(start.z != end.z || start == end ) //no pathfinding between z levels
```

Comment:
Should still not permit cases of start = end.

Reply:
true!